src/snyk/common/vscode/editorDecorator.ts

```diff
@@ -0,0 +1,36 @@
+import { IThemeColorAdapter } from './theme';
```

Comment:
`vscode` folder contains no logic but interfacing to `vscode` API/module. `editorDecorator.ts` module contains logic currently with a specific implementation. 

My suggestion: move this file to `src/snyk/common/editor/editorDecorator.ts`.

Reply:
It's `outdated`.